packages/webservice/app/src/test/kotlin/com/yahoo/navi/ws/test/integration/StarWarsTest.kt

```diff
@@ -0,0 +1,20 @@
+package com.yahoo.navi.ws.test.integration
+
+import com.jayway.restassured.RestAssured.given
+import com.yahoo.navi.ws.test.framework.IntegrationTest
+import org.hamcrest.Matchers
+import org.junit.jupiter.api.Test
+
+class StarWarsTest : IntegrationTest() {
```

Comment:
Can we rename this file to `DemoDataSourceTest.kt`

Reply:
changed.